variables.tf

```diff
@@ -118,6 +118,24 @@ variable "instance_count" {
   default     = 4
 }
 
+variable "warm_enabled" {
+  type        = bool
+  default     = false
+  description = "Whether AWS UltraWarm is enabled"
+}
+
+variable "warm_count" {
+  type        = number
+  default     = 2
```

Comment:
I'm wondering, should we even bother with a `warm_enabled` flag? Could we just state warm_count > 0? I'm wondering if the module should hide the relationship between the two settings or act as a very thin shim and expose it.

> warm_enabled - (Optional) Indicates whether to enable warm storage.
> warm_count - (Optional) The number of warm nodes in the cluster. Valid values are between 2 and 150. warm_count can be only and must be set when warm_enabled is set to true.

Reply:
@deiga we usually provide all the variables that a resources has, so having both in this case is ok
please rebase, and thanks for the PR